source/ada/lsp-ada_highlighters.adb

```diff
@@ -0,0 +1,110 @@
+------------------------------------------------------------------------------
+--                         Language Server Protocol                         --
+--                                                                          --
+--                     Copyright (C) 2018-2022, AdaCore                     --
```

Comment:
This file is just added, so it can't contains 2018

Reply:
I've copied part of it from GPS sources, where it was since 2018. But I'll change if you want